jsk_spot_robot/jsk_spot_startup/launch/jsk_spot_bringup.launch

```diff
@@ -3,6 +3,8 @@
   <arg name="password" default="dummypassword" />
   <arg name="hostname" default="192.168.50.3" />
 
+  <arg name="pad_type" default="R1" /> <!-- Option: dualshock3, R1 -->
```

Comment:
this arg is not used?

Reply:
No, not used. I notice it is redundant. thanks.